samwise.js

```diff
@@ -6,100 +6,100 @@
  */
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.